### DISCUSSION TITLE
1.19.2 fabric #179 d778245 部分未添加配方

### DISCUSSION THREAD

**Discussion by FrozenAshRfg** [IndustrialCraftMC/IndustrialCraft2#113] [【漏洞反馈】Bugs Report]
1. 基本信息：
  minecraft 1.19.2 Fabric 0.14.22
  AMD Ryzen 5 4650G； 6700XT 12G（23.7.2）； 32G*2 DDR4 3200MHZ

2. 相关模组：
(1) 实验组（1.19.2 fabric）
     ic2-fabric-unspecified+d778245.jar
     jei-1.19.2-fabric-11.6.0.1018.jar
(2) 对照组（1.12.2 forge）
    industrialcraft-2-2.8.221-ex112.jar
     jei_1.12.2-4.16.1.1000.jar

3. 部分缺失配方对照表
![209e2a8a436da42b444788d08eb5ab85](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/108665535/6468f48e-c72d-4ba3-9c9a-a23d78fa5745)

**Comment by AbbottMc** [comment]
非常好反馈，爱来自开发者
